### PR TITLE
Replace is_host_win with host_os == "win" to better match upstream chromium style

### PR DESCRIPTION
**Pull request by niranjanyardi**
b/319086452

Replace is_host_win with host_os == "win" to match chromium's style of identifying if host os is windows.

### CODE REVIEW

File: third_party/perfetto/gn/standalone/BUILDCONFIG.gn [hunk 1]
```diff
@@ -31,14 +31,7 @@ if (current_os == "") {
...
 
-is_android = current_os == "android"
-is_chromeos = current_os == "chromeos"
-is_linux = current_os == "linux"
-is_linux_host = host_os == "linux"
-is_mac = current_os == "mac"
-is_mac_host = host_os == "mac"
-is_win = current_os == "win"
-is_win_host = host_os == "win"
+import("starboard/build/config/os_definitions")
```
Comment (andrewsavage1) [inline third_party/perfetto/gn/standalone/BUILDCONFIG.gn:34]: Wrap with `use_cobalt_customizations`

Comment (niranjanyardi) [reply third_party/perfetto/gn/standalone/BUILDCONFIG.gn]: fixed

File: third_party/perfetto/gn/standalone/BUILDCONFIG.gn [hunk 2]
```diff
@@ -31,14 +31,9 @@ if (current_os == "") {
...
 
-is_android = current_os == "android"
-is_chromeos = current_os == "chromeos"
-is_linux = current_os == "linux"
-is_linux_host = host_os == "linux"
-is_mac = current_os == "mac"
-is_mac_host = host_os == "mac"
-is_win = current_os == "win"
-is_win_host = host_os == "win"
+if (use_cobalt_customizations) {
```
Comment (andrewsavage1) [inline third_party/perfetto/gn/standalone/BUILDCONFIG.gn:34]: This is exactly what it should look like:

```
if (use_cobalt_customizations) {
  import("starboard/build/config/os_definitions")
} else {
is_android = current_os == "android"
is_chromeos = current_os == "chromeos"
is_linux = current_os == "linux"
is_linux_host = host_os == "linux"
is_mac = current_os == "mac"
is_mac_host = host_os == "mac"
is_win = current_os == "win"
is_win_host = host_os == "win"
}
```

**Comment by niranjanyardi** [comment]
Test breakage is unrelated, filed a separate bug: b/348701290

**Comment by niranjanyardi** [comment]
talked to kaido offline , seems like upstream code uses `host_os = "win"  ` directly in most cases . The idea is to match upstream code as much as possible , so we'll end up removing is_win_host, is_linux_host, is_mac_host  from os_definitions.gni  .

Does that sg @andrewsavage1 ?

**Comment by andrewsavage1** [comment]
> talked to kaido offline , seems like upstream code uses `host_os = "win" ` directly in most cases . The idea is to match upstream code as much as possible , so we'll end up removing is_win_host, is_linux_host, is_mac_host from os_definitions.gni .
> 
> Does that sg @andrewsavage1 ?

That's fine, it doesn't make much of a difference either way

**Review by andrewsavage1 — APPROVED** [review]

**Comment by niranjanyardi** [comment]
doublechecked that the variable isn't used in internal builds - so shouldnt break internal builds

**Pull request merged**